src/js/pages/services/task-details/TaskDetailsTab.js

```diff
@@ -9,6 +9,14 @@ import TaskDirectoryStore from '../../../stores/TaskDirectoryStore';
 import Units from '../../../utils/Units';
 
 class TaskDetailsTab extends React.Component {
+  getDescriptionLists(task) {
+    return [
+      this.getMesosTaskDetailsDescriptionList(task),
+      this.getMesosTaskLabelDescriptionList(task),
+      <MarathonTaskDetailsList taskID={task.id} />
```

Comment:
@daiyiwhale did you miss pushing up a change to add the `key` or am I missing something?